src/GeneratedHydrator/CodeGenerator/Visitor/HydratorMethodsVisitor.php

```diff
@@ -73,10 +71,15 @@ public function __construct(ReflectionClass $reflectedClass)
      */
     public function leaveNode(Node $node)
     {
-        if (! $node instanceof Class_) {
+        if (!$node instanceof Class_) {
             return null;
         }
 
+        $node->stmts[] = new Property(Class_::MODIFIER_PRIVATE, array(
```

Comment:
Once again, just keeping consistency with the rest of the code. I am on your side with this, I'd be please to use the short array syntax, but once again, I prefer to have @Ocramius opinion before changing any code convention this library uses.